src/crypto/index.ts

```diff
@@ -1845,6 +1845,29 @@ export class Crypto extends TypedEventEmitter<CryptoEvent, CryptoEventHandlerMap
         // the backup with the new key (if not already signed)?
     }
 
+    /**
+     * Implementation of {@link CryptoBackend#getBackupDecryptor}.
+     */
+    public async getBackupDecryptor(backupInfo: IKeyBackupInfo, privKey: ArrayLike<number>): Promise<BackupDecryptor> {
+        if (!(privKey instanceof Uint8Array)) {
+            // eslint-disable-next-line @typescript-eslint/no-base-to-string
+            throw new Error(`getBackupDecryptor expects Uint8Array, got ${privKey}`);
```

Comment:
Is it potentially dangerous to log the private key here? Maybe we could log its type, or omit it.

Reply:
(would also mean we could remove the `eslint-disable...`

Reply:
Right, was copied from old code but agree that it doesn't look right. Removed